madgraph/iolibs/export_v4.py

```diff
@@ -6984,7 +6992,7 @@ def create_coupl_inc(self):
                 double precision MU_R, all_mu_r%(vec)s
                 common/rscale/ MU_R, all_mu_r
 
-                """   % {'vec': ("(%i)" % max(1,self.vector_size) if self.vector_size else '')}
+                """   % {'vec': ("(VECSIZE_MEMMAX)" if self.vector_size else '')}
 
```

Comment:
Can we keep the max here ?

Reply:
hm I need to think about that... what is the max used for?

Reply:
I do not fully remember to be honest it was introduce to be able to catch specific case where vectorization is not used.

Reply:
Ok I wil try to have a look

Reply:
Hi Olivier, I am not really sure what the logic was, but I changed it so that it is "1" hardcoded if vector_size <=1,
https://github.com/mg5amcnlo/mg5amcnlo/pull/23/commits/066fb101243a81179b29d6144306ae94de971cdf
Hope this can be resolved?